Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use OpenShift golang builder image
 # These images needs to be synced with the images in the Makefile.
-ARG BUILDER_IMAGE=${BUILDER_IMAGE:-registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13}
+ARG BUILDER_IMAGE=${BUILDER_IMAGE:-registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.13}
```

Comment:
does it makes sense to use the following images ?
BUILDER_IMAGE=registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
TARGET_IMAGE=registry.ci.openshift.org/ocp/4.16:base-rhel9

Ref: https://github.com/openshift/release/blob/077ae802a5461f63a34ba5157cc2bc2783db7219/ci-operator/config/openshift/oc/openshift-oc-release-4.16.yaml

Reply:
These are  likely the images @spotlesstofu is looking for. Thanks @bpradipt !